packages/blade/src/components/BladeProvider/stylisCSSHigherSpecificity.ts

```diff
@@ -0,0 +1,13 @@
+import type { StylisPlugin } from 'styled-components';
+
```

Comment:
did you referf this plugin? can you check how is this different than ours in terms of behaviour?
https://github.com/sanusart/stylis-plugin-extra-class-names-specifity/blob/main/index.js

Reply:
@kamleshchandnani this seems to be doing the same thing, this seems to have some additional checks on [`@keyframe`](https://github.com/sanusart/stylis-plugin-extra-class-names-specifity/blob/2cc91d4cd2253fdfeb2e9e27d8065d4fc632f9a9/test.js#L18-L20) and [`nested/chained` styles](https://github.com/sanusart/stylis-plugin-extra-class-names-specifity/blob/2cc91d4cd2253fdfeb2e9e27d8065d4fc632f9a9/test.js#L22).

Also, this plugin is using stylis v4, hence the syntax is different. [styled-component v6 or later](https://styled-components.com/releases#v6.0.0) is required to use stylis v4.